docs/rules/interpolate-not-concat.md

```diff
@@ -0,0 +1,29 @@
+# Interpolation preferred
+
+**Code**: interpolate-not-concat
```

Comment:
Maybe the code should be prefer-interpolation?   There are rules like that in typescript lint (e.g. https://eslint.org/docs/rules/prefer-template)